tasks/github_tasks.py

```diff
@@ -66,28 +80,36 @@ def trigger_macos_test(
     retry_download=3,
     retry_interval=10,
 ):
-    env = load_release_versions(ctx, release_version)
-    github_action_ref = env["MACOS_BUILD_VERSION"]
-
-    run = trigger_macos_workflow(
+    _trigger_macos_workflow(
+        release_version,
+        destination,
+        retry_download,
+        retry_interval,
         workflow_name="test.yaml",
-        github_action_ref=github_action_ref,
         datadog_agent_ref=datadog_agent_ref,
         python_runtimes=python_runtimes,
         version_cache_file_content=version_cache,
     )
 
-    workflow_conclusion, workflow_url = follow_workflow_run(run)
-
-    if workflow_conclusion == "failure":
-        print_failed_jobs_logs(run)
-
-    print_workflow_conclusion(workflow_conclusion, workflow_url)
-
-    download_with_retry(download_artifacts, run, destination, retry_download, retry_interval)
 
-    if workflow_conclusion != "success":
-        raise Exit(code=1)
+@task
+def trigger_macos_lint(
+    _,
+    datadog_agent_ref=DEFAULT_BRANCH,
+    release_version="nightly-a7",
+    python_runtimes="3",
+    version_cache=None,
+):
+    _trigger_macos_workflow(
+        release_version,
+        None,
+        0,
+        0,
```

Comment:
❓ Question :
Maybe worth putting these args as default value ?

Reply:
I wasn't sure how mixing default values & kwargs would behave but on a second thought I don't really see what the problem could be, I'll make the change